src/utils/sources-tree/treeOrder.js

```diff
@@ -93,6 +98,37 @@ function createTreeNodeMatcherWithDebuggeeHost(
   };
 }
 
+function createTreeNodeMatcherWithBundler(
+  debuggeeHost: ?string
+): FindNodeInContentsMatcher {
+  return (node: TreeNode) => {
+    if (node.name === IndexName) {
+      return -1;
+    }
+    if (debuggeeHost && isExactDomainMatch(node.name, debuggeeHost)) {
+      return -1;
+    }
+    return isBundler(node.name) ? 0 : 1;
```

Comment:
One edge case:
We call `createTreeNodeMatcher` function in line 174 with `part = "ng://"`
Therefore, line 191 `isBundler(part)` will evaluate to true and we will return `createTreeNodeMatcherWithBundler`

Then, we call `createTreeNodeMatcherWithBundler` where `node.name = "webpack://"`
Therefore, line 111 `isBundler(node.name)` will evaluate to true and we will return 0

However, this is not the correct value to return because webpack and angular are not the same and it should not return 0 because 0 means they are the same. Let me know if this makes sense

Reply:
> @darkwing Yes, it results in one of the bundlers group with the other!

Yes I could not do this earlier because I thought the same, But I think these changes do no effect the tree order/directories, let me confirm again, then I will commit the changes. May be I need to revert this back